openkfw/TruBudget
# Additional configuration of nginx from env vars

### Checklist




### Description

Option for admin to add custom configuration of nginx server

### How to test

1. Check if e2e tests passed
2. Locally change Dockerfile.dev to Dockerfile in development docker-compose
3. Modify the new env variables
4. After starting of app with development/start-dev.sh script (note that frontend will not work) validate that nginx config was changed properly
5. you can use `docker exec -it trubudget-dev-frontend-1 sh` command and then display nginx config




Closes #1976
